fix for error: no suitable conversion function from "std::istream" to "bool" exists
author: Kenneth Hoste (HPC-UGent)
--- libgtextutils-0.7/src/gtextutils/text_line_reader.cpp.orig	2018-06-08 17:38:32.090881776 +0200
+++ libgtextutils-0.7/src/gtextutils/text_line_reader.cpp	2018-06-08 17:38:55.261151439 +0200
@@ -44,6 +44,6 @@
 	if (input_stream.eof())
 		return false;
 
-	return input_stream ;
+	return (bool)input_stream ;
 }
 
